From 65e4ce3de9747e359f38db65cf78763f75bcfbd1 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?J=C3=B8rgen=20Nordmoen?= <nordmoen@users.noreply.github.com>
Date: Fri, 4 May 2018 15:50:39 +0200
Subject: [PATCH] Changed invocation to `add` to conform template syntax
 (#1388)

This change fixes issue #1383

Upstream-Status: Backport [from melodic, https://github.com/ros/ros_comm/commit/65e4ce3de9747e359f38db65cf78763f75bcfbd1]

This patch has been generated with:
  `git format-patch -1 65e4ce3de9747e359f38db65cf78763f75bcfbd1`
in the ros_comm repository.

Signed-off-by: Lukas Bulwahn <lukas.bulwahn@gmail.com>
---
 utilities/message_filters/include/message_filters/synchronizer.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/utilities/message_filters/include/message_filters/synchronizer.h b/utilities/message_filters/include/message_filters/synchronizer.h
index 7891890..1c14a6f 100644
--- a/utilities/message_filters/include/message_filters/synchronizer.h
+++ b/utilities/message_filters/include/message_filters/synchronizer.h
@@ -355,7 +355,7 @@ private:
   template<int i>
   void cb(const typename mpl::at_c<Events, i>::type& evt)
   {
-    this->add<i>(evt);
+    this->template add<i>(evt);
   }
 
   uint32_t queue_size_;
-- 
2.7.4

